### DISCUSSION TITLE
Is it possible to get all upcoming calendar event in one email by workflow?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16932] [How to]
Hello
I created a workflow which will send an email to the assigned user for the upcoming calendar activity for today.
But If he/she has more than 1 activity for the day, he will get as many emails as events he/she has.

Also the same for the overdue activities, if he/she has 10 overdue activity, he/she will get 10 mail.

Is it possible to get 1 email with more than 1 calendar item?

Thank you

**Comment by rskrzypczak** [comment]
Yes, using reports (email templates with appropriate functions) in the workflow and schedule.

![image](https://user-images.githubusercontent.com/10221999/222062200-12f684fe-687e-4d08-a7f1-a4e29e1fff52.png)
![image](https://user-images.githubusercontent.com/10221999/222064522-0ce0a9ce-233e-4f7a-8059-53c4eaa94cdc.png)

and in email template set e.g.: `$(custom : UserCalendar|today)$`

**Reply by JEEVAN28** [reply]
Email Report option is not showing in 6.4

**Reply by rskrzypczak** [reply]
I have to disagree.
The checkbox visible in the screenshot is important for the availability of this function
